tests/integration/test_terraform.py

```diff
@@ -65,3 +65,28 @@ def test_lambda(self):
         self.assertEqual(response['Configuration']['Handler'], LAMBDA_HANDLER)
         self.assertEqual(response['Configuration']['Runtime'], LAMBDA_RUNTIME)
         self.assertEqual(response['Configuration']['Role'], LAMBDA_ROLE)
+
+    def test_apigateway(self):
+        apigateway_client = aws_stack.connect_to_service('apigateway')
+        rest_apis = apigateway_client.get_rest_apis()
+
+        for rest_api in rest_apis['items']:
+            if rest_api['name'] == 'test-tf-apigateway':
+                rest_id = rest_api['id']
+                continue
```

Comment:
nit: I guess this should be `break` instead (shouldn't hold back the merge, though)